fastly/terraform/dev_override.tf

```diff
@@ -40,7 +40,7 @@ resource "fastly_service_v1" "app" {
     first_byte_timeout    = 120000
     between_bytes_timeout = 120000
     error_threshold       = 0
-    shield                = "dca-dc-us"
+    shield                = "iad-va-us"
```

Comment:
This change had to be include because DCA no longer exists - the original PR for the DCA change was https://github.com/Financial-Times/polyfill-service/pull/2659